xds/src/main/java/io/grpc/xds/XdsEndpointResource.java

```diff
@@ -101,7 +100,8 @@ protected EdsUpdate doParse(Args args, Message unpackedMessage) throws ResourceI
   }
 
   private static boolean isEnabledXdsDualStack() {
-    return GrpcUtil.getFlag(GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS, false);
+    return GrpcUtil.getFlag(PickFirstLoadBalancerProvider.GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS,
```

Comment:
We will need to be careful here to not flip the flag before it is passing interop. We can flip the happy eyeballs flag without interop passing, but we shouldn't flip this one.